### DISCUSSION TITLE
some fontawesome icons cannot be displayed

### DISCUSSION THREAD

**Discussion by imnanxu** [greenelab/lab-website-template#279] [Q&A] [answered]
### Checks

- [X] I have searched **[the docs](https://greene-lab.gitbook.io/lab-website-template-docs)**, [existing issues](https://github.com/greenelab/lab-website-template/issues), and [existing discussions](https://github.com/greenelab/lab-website-template/discussions) for answers first.

### Link to your website repo

https://github.com/inspirelab-site/inspirelab.site

### Version of Lab Website Template you are using

1.2.2

### Description

Hi, I used the fa-person-chalkboard icon for PI, and have updated the latest release of fontawesome: {% assign fontawesome = "https://use.fontawesome.com/releases/v6.5.0/css/all.css" %}. However, I realized that it cannot be displayed on the website.. I wonder how to fix it?
![icon_err](https://github.com/user-attachments/assets/3e72a8fa-a968-40e8-baba-7cc3d0f19365)

**Comment by vincerubinetti** [comment]
It looks like you might've solved this by yourself already, I can see it here:

<img src="https://github.com/user-attachments/assets/a99d0b57-2f78-48d0-aebc-99891698a1af" height="300"/>

If so, could you share what you did so other people can benefit from the knowledge.

Also at the moment, 6.6.0 is the latest version.

**Reply by vincerubinetti** [reply, marked as answer]
Okay, very strangely, when I was looking at your site just an hour ago on my phone, the icon appeared, as shown in my screenshot. But now that I'm on my desktop, I don't see it.

What I can tell you is that the icon is actually `fa-solid fa-person-chalkboard`, not `fa-regular` as you have it in your [source code](https://github.com/search?q=repo%3Ainspirelab-site%2Finspirelab.site%20fa-person-chalkboard&type=code). 

The solid vs. regular vs. whatever are the different "styles" of the same icons they offer. But confusingly, some icons are free in one style but require a pro license in another. This icon is one of those:

<img src="https://github.com/user-attachments/assets/c4eef210-ae54-4edc-acd0-7c9dd62a6675" height="200">

I can't explain why I was able to see it on my phone. Looking at the screenshot, that's definitely the free `fa-solid` (thicker, filled-in) style, so somehow it must've been able to fall back to that style, just on my phone, somehow.

**Reply by imnanxu** [reply]
Thank you @vincerubinetti ! Yes, I changed to "fa-solid fa-person-chalkboard" and it worked!
